__tests__/errorReporting.spec.js

```diff
@@ -488,15 +489,20 @@ describe('error reporting', () => {
   describe('serverSideError', () => {
     it('should log the error', () => {
       expect.assertions(1);
-      const err = new Error('this is a test');
-      return serverSideError(err).then(() => {
-        expect(consoleErrorSpy).toHaveBeenCalledWith(util.inspect(err, false, 10, true));
+      const queue = [{
+        type: 'ServerSideReportedError',
+        msg: 'test error',
+        stack: '1\n2\n3',
+      }];
+      const err = new Error('test error');
+      return serverSideError(queue).then(() => {
+        expect(consoleErrorSpy).toHaveBeenCalledWith(err);
```

Comment:
I'm a bit confused, sorry: if `err` isn't in the queue, how would console.error have been invoked with it?

Reply:
Yeah this error is flaky, will fix it when we decide about the `name` or `type` property

Reply:
fixed https://github.com/americanexpress/one-app-ducks/pull/32/commits/fbf50559d7f297c2486b7034af3ece41275187a5